src/api/Feed.js

```diff
@@ -401,7 +401,13 @@ class Feed extends Base {
         this.updateFeedItem({ isPending: true }, task.id);
 
         try {
-            await Promise.all(task.addedAssignees.map(assignee => this.createTaskCollaborator(file, task, assignee)));
+            await Promise.all(
+                task.addedAssignees.map(assignee =>
```

Comment:
See https://github.com/box/box-ui-elements/pull/1809/commits/6f02ea3c97c8ccd4886516017e51b5a9be20e241#diff-96176db00fd4ba294b0258debdbd3a25R665 Last week, we chose to check specifically for the two types and reject if some unknown type is encountered.

Since these two should be identical (correct?) I think one of them needs changing. I lean toward the above, as it'll fail in a useful way. We could also `.reject` early to avoid attempting to write known-bad collaborators.